packages/expo-modules-core/ios/TestUtils/TestingJSCallInvoker.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#pragma once
+
+#ifdef __cplusplus
+
+#include <ReactCommon/CallInvoker.h>
+#include "MainThreadInvoker.h"
+
+#include <jsi/jsi.h>
+
+namespace jsi = facebook::jsi;
+namespace react = facebook::react;
+
+namespace expo {
+
+/**
+ * Dummy CallInvoker.
+ * Used in the test environment to check the async flow.
```

Comment:
update the comment to describe the difference between this class and the existing `TestingSyncJSCallInvoker`: https://github.com/expo/expo/blob/196df28c2f650318101ba6417c45e477fd57adeb/packages/expo-modules-core/common/cpp/TestingSyncJSCallInvoker.h#L16-L20